src/Stratis.FederatedPeg.Features.FederationGateway/Interfaces/ICrossChainTransfer.cs

```diff
@@ -44,7 +48,7 @@ public interface ICrossChainTransfer : IBitcoinSerializable
         /// <summary>
         /// The height of the block where the transaction resides on our chain.
         /// </summary>
-        int BlockHeight { get; }
+        int? BlockHeight { get; }
```

Comment:
same here, when is it meant to be null, I assume on suspended only ?

Reply:
The block height will be null whenever a transfer's transaction has not made it to the B chain yet. Don't confuse this with the `DepositHeight`. The latter an be null when a block is seen on chain B before the corresponding deposit information is received.

Reply:
yes, being silly here, thanks !